libraries/botbuilder-core/src/turnContext.ts

```diff
@@ -420,7 +420,7 @@ export class TurnContext {
             const o: Partial<Activity> = TurnContext.applyConversationReference({...a}, ref);
             if (!o.type) { o.type = ActivityTypes.Message; }
             if (o.type !== ActivityTypes.Trace) { sentNonTraceActivity = true; }
-
+            if (o.id) { delete o.id; }
```

Comment:
Can you add a test for this? The bot sends the same activity twice and the id should be different on both activities.